client/implementation-vertx/pom.xml

```diff
@@ -33,5 +42,11 @@
             <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-junit-jupiter</artifactId>
+            <version>${version.mockito}</version>
+            <scope>compile</scope>
```

Comment:
Should this not be test ?

Reply:
Good catch, I copied that over from https://github.com/smallrye/smallrye-graphql/blob/1.2.8/client/implementation-jaxrs/pom.xml#L41 including the mistake... Fixed now (in this PR, not in upstream:) )

Reply:
Cool